pkg/cloud-provider/cloudapi/aws/aws_security.go

```diff
@@ -115,6 +115,11 @@ func (ec2Cfg *ec2ServiceConfig) createOrGetSecurityGroups(vpcID string, cloudSgN
 		}
 	}
 
+	// return the up-to-date cloud objects for SGs
+	if len(cloudSgNamesToCreate) == 0 {
```

Comment:
what is this? should this be part of this PR

Reply:
Re-checked it. There were 2 instances of this snippet. So removed one.